check-ecs-exec.sh

```diff
@@ -122,7 +122,8 @@ AWS_REGION=${AWS_REGION:-$REGION}
 
 callerIdentityJson=$(${AWS_CLI_BIN} sts get-caller-identity)
 ACCOUNT_ID=$(echo "${callerIdentityJson}" | jq -r ".Account")
-MY_IAM_ARN=$(echo "${callerIdentityJson}" | jq -r ".Arn")
+MY_IAM_ARN=$(echo "${callerIdentityJson}" | jq -r '.Arn |= sub("assumed-role"; "role") | .Arn')
+MY_IAM_ARN="${MY_IAM_ARN%\/*}"
```

Comment:
Is this safe for normal ARNs?

Reply:
:facepalm: you're right. Simplified it by using everything before the second occurrence of slash in the ARN, as that seems to be the thing that chokes simulate-principal-policy.